app/cli/cmd/attestation_push.go

```diff
@@ -64,7 +65,10 @@ func newAttestationPushCmd() *cobra.Command {
 			a, err := action.NewAttestationPush(&action.AttestationPushOpts{
 				ActionsOpts: actionOpts, KeyPath: pkPath, BundlePath: bundle,
 				CLIVersion: info.Version, CLIDigest: info.Digest,
-				SignServerCAPath: signServerCAPath,
+				SignServerCAPath:   signServerCAPath,
+				CASURI:             viper.GetString(confOptions.CASAPI.viperKey),
```

Comment:
Why don't we just pass the CAS connection? We have already code in the CMD package to generate the connection https://github.com/migmartri/chainloop/blob/40f96e6edc5c4f3bba347b30b44af88614dd8e30/app/cli/cmd/artifact.go#L38-L62

you can see an example in artifact_download command

Reply:
We can't, because CAS connection is specific for a given digest, and here we wanted to download "any" material, so we need to create specific connections for each.